### DISCUSSION TITLE
Request: Add information from reprints to TPB

### DISCUSSION THREAD

**Discussion by bpepple** [bpepple/metron#213] [General]
If you want the admin to add reprint information for TPB, you can make a request here. Please verify that the issues reprinted in the TPB have been added and the just add message in this discussion in the following format:

- [ ] {Trade Paperback Name} #{Volume Number}

**Comment by BasicMiniTacos** [comment]
East of West #2
Let me know if I missed any core info you need

**Comment by bpepple** [comment]
> East of West #2 Let me know if I missed any core info you need

Looks like the issues reprinted in the TPB weren't added (which is what is used to pull characters, story titles, etc). You can add the issues in the `Reprint` field.

Also, the ISBN and UPC fields shouldn't contain hyphens. I've added a [bug](https://github.com/bpepple/metron/issues/215) report to add validations to the `Issue` form for these fields.

**Reply by BasicMiniTacos** [reply]
Should be fixed

**Reply by bpepple** [reply]
Ok, I added the info from the admin command. Looks like the only info that's missing is the TPB summary and the creator credits.

**Reply by BasicMiniTacos** [reply]
Is there a reason creator credits weren't pulled for this volume? They were for the first one

**Reply by bpepple** [reply]
I added those manually.

Trying to get creators & their roles merge into usable data isn't particularly easy, and a lot of the time it differs from the TPB listed credits, so I don't bother adding them with an admin command.

**Comment by BasicMiniTacos** [comment]
Ice Cream Man 1, 2, 3, and 4

**Reply by bpepple** [reply]
Done.

**Comment by BasicMiniTacos** [comment]
Ice Cream Man  5, 6, 7, 8, 9 TPB

**Reply by bpepple** [reply]
Done.

**Comment by bpepple** [comment]
Collections that need to have reprint information added:

- [ ] Edge of Spider-Geddon TPB `#1`
- [x] The Flash TPB `#1`
- [x] The Flash TPB `#2`
- [x] The Flash TPB `#3`
- [x] The Flash TPB `#4`
- [x] The Flash TPB `#5`
- [x] The Flash TPB `#6`
- [x] The Flash TPB `#7`
- [ ] The Flash TPB `#8`
- [ ] The Flash TPB `#9`
- [ ] The Flash TPB `#10`
- [ ] The Flash TPB `#11`
- [ ] The Flash TPB `#12`
- [ ] The Flash TPB `#13`
- [ ] The Flash TPB `#14`
- [ ] The Flash TPB `#15`
- [x] The Flash TPB `#16`
- [ ] The Flash TPB `#17`
- [ ] The Flash TPB `#18`
- [x] The Flash TPB `#19`
- [ ] The Flash TPB `#20`
- [x] Magic HC `#1`
- [x] Magic HC `#2`
- [x] Magic HC `#3`
- [ ] Magic HC `#4`
- [ ] Magic HC `#5`
- [ ] Peter Parker: The Spectacular Spider-Man TPB `#1`
- [ ] Peter Parker: The Spectacular Spider-Man TPB `#2`
- [ ] Peter Parker: The Spectacular Spider-Man  `#1`
- [ ] Peter Parker: The Spectacular Spider-Man TPB `#3`
- [ ] Peter Parker: The Spectacular Spider-Man TPB `#4`
- [ ] Peter Parker: The Spectacular Spider-Man TPB `#5`
- [ ] Shazam! TPB `#1`
- [ ] Spider-Geddon TPB `#1`
- [x] Spider-Geddon: Covert Ops TPB `#1`
- [ ] Spider-Island TPB `#1`
- [ ] Spider-Man: Life Story TPB `#1`
- [ ] Symbiote Spider-Man TPB `#1`
- [ ] Symbiote Spider-Man: Alien Reality TPB `#1`
- [ ] Symbiote Spider-Man: Crossroads TPB `#1`
- [ ] Symbiote Spider-Man: King In Black TPB `#1`

**Comment by Buried-In-Code** [comment]
- [x] Spidey TPB `#1`

**Reply by bpepple** [reply]
Added this. This one looks like it was the 2nd TPB for this series and seems to be consistently named as `Spidey: Freshman Year`. 

Here the links to how other sources named it:

- [GCD](https://www.comics.org/issue/1943876/)
- [LOCG](https://leagueofcomicgeeks.com/comic/6693452/spidey-freshman-year-tp)
- [Marvel](https://www.marvel.com/comics/issue/106561/spider-man_freshman_year_trade_paperback)

**Reply by Buried-In-Code** [reply]
Ah right, I hadn't come across ths others, but yes looks like they reprinted it in 2 types the first set being 1-6 and then 7-12 and then this TP is 1-12

**Comment by Buried-In-Code** [comment]
- [x] [You Look Like Death: Tales From the Umbrella Academy TPB](https://metron.cloud/issue/you-look-like-death-tales-from-the-umbrella-academy-2021-1/)

**Comment by Buried-In-Code** [comment]
- [x] [Powers TPB 1-6](https://metron.cloud/series/powers-2022/)

**Comment by Buried-In-Code** [comment]
- [x] [The Amazing Spider-Man Epic Collection 27](https://metron.cloud/issue/the-amazing-spider-man-epic-collection-2013-27/)
- [x] [The Amazing Spider-Man Epic Collection 28](https://metron.cloud/issue/the-amazing-spider-man-epic-collection-2013-28/)

**Comment by Buried-In-Code** [comment]
- [x] [Cloak and Dagger v1 TPB](https://metron.cloud/series/cloak-and-dagger-2017/)
- [x] [Cloak and Dagger v2 TPB](https://metron.cloud/series/cloak-and-dagger-2017-1/)
- [x] [Cloak and Dagger v3 TPB](https://metron.cloud/series/cloak-and-dagger-2018-1/)
- [x] [Cloak and Dagger v4 TPB](https://metron.cloud/series/cloak-and-dagger-2018-2/)
- [ ] [Cloak and Dagger v5](https://metron.cloud/series/cloak-and-dagger-2018-4/) *Not a TPB but collects digital*
- [x] [Cloak and Dagger v5 TPB](https://metron.cloud/series/cloak-and-dagger-2018-3/)
- [x] [Cloak and Dagger v6 TPB](https://metron.cloud/series/cloak-and-dagger-2019/)

I also added but ran out of time to properly fill it in, so might need a set of eyes [Strange Tales: Dark Corners](https://metron.cloud/issue/strange-tales-dark-corners-1998-1/) according to CV it should be a **Strange Tales** imprint of Marvel not a direct marvel, however none of the other Strange Tales are an imprint.

**Reply by bpepple** [reply]
Did a little research and couldn't find anything about a **Stange Tales** imprint, so I don't think it makes much sense to add one.

**Comment by Buried-In-Code** [comment]
- [x] [Miles Morales: Spider-Man v2 TPB #2](https://metron.cloud/issue/miles-morales-spider-man-2023-1-2/)
- [x] [Miles Morales: Spider-Man v2 TPB #3](https://metron.cloud/issue/miles-morales-spider-man-2023-1-3/)
- [x] [Miles Morales: Spider-Man v2 TPB #4](https://metron.cloud/issue/miles-morales-spider-man-2023-1-4/)
- [ ] [Spider-Man v2 TPB](https://metron.cloud/series/spider-man-2016-1/)
- [ ] [Spider-Man v3 TPB](https://metron.cloud/series/spider-man-2021/)
- [ ] [Spider-Man v4 TPB](https://metron.cloud/series/spider-man-2023/)
- [x] [Spider-Man/Spider-Gwen TPB](https://metron.cloud/series/spider-manspider-gwen-2017/)
